### DISCUSSION TITLE
How to get Yolo badge

### DISCUSSION THREAD

**Discussion by kaji98** [community/community#19987] [Other Feature Feedback, Questions, & Ideas] [answered]
hi everyone, i'm new on github
I just got the pull shark badge, how do i get the yolo badge? even though it merges without any review
![image](https://user-images.githubusercontent.com/108684003/177188229-4c0c656e-5bb5-4e6c-8b6d-fa495fadfaa9.png)

**Comment by vietdoo** [comment, marked as answer]
hi @kaji98  😀

You need follow the below steps.

    _```
Create a pull request on your own remote repository.
    Open the pull request.
    At the top right corner add a reviewer to your repository.
    Before they review your code and merge you should merge your code to the remote repository.
```_

hope the answer can help you.
